packages/storybook/src/stories/ino-option-group/ino-option-group.stories.ts

```diff
@@ -1,12 +1,13 @@
+import { Meta } from '@storybook/web-components';
 import { html, TemplateResult } from 'lit-html';
-import { defaultDecorator } from '../utils';
+import { decorateStoryWithClass } from '../utils';
 import './ino-option-group.scss';
 
 export default {
   title: 'Input/ino-option-group',
   component: 'ino-option-group',
-  decorators: [defaultDecorator],
-};
+  decorators: [(story) => decorateStoryWithClass(story, 'story-option-group')],
```

Comment:
The does css class `story-option-group-customizable` wrapped in `story-option-group` too? Currently only `ino-select` is wrapped.

Reply:
Whops, my bad. Should be fixed now.